lib/ReactViews/Credits/MapCredits/MapCredits.tsx

```diff
@@ -0,0 +1,71 @@
+import { reaction } from "mobx";
+import { observer } from "mobx-react";
+import React, { FC, useCallback, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import GlobeOrMap from "../../../Models/GlobeOrMap";
+import { ICredit } from "../Credit.type";
+import { Credits } from "../Credits";
+import { CreditsContainer } from "../CreditsContainer";
+import { DataAttributionModal } from "../DataAttribution/DataAttributionModal";
+import { Spacer } from "../Spacer";
+import { TerriaLogo } from "../TerriaLogo";
+import { MapCreditLogo } from "./MapCreditLogo";
+
+interface IMapCreditsProps {
+  hideTerriaLogo: boolean;
+  credits?: ICredit[];
+  currentViewer: GlobeOrMap;
+}
+
+export const MapCredits: FC<IMapCreditsProps> = observer(
+  ({ currentViewer, hideTerriaLogo, credits }) => {
+    const { t } = useTranslation();
+    const [dataAttributionVisible, setDataAttributionVisible] = useState(false);
+
+    const showDataAttribution = useCallback(() => {
```

Comment:
I see you are the only contributor to use useCallback() perhaps we could make more use of it to make things more efficient?